packages/ui/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 ## [unreleased]
 
+### Fixed
+
+- `ui/ScrollableNative`, `ui/Scroller`, and `ui/VirtualListBase` to scroll correctly on iOS and Safari
```

Comment:
Reminder that `ui/Scrollable` is private and `ui/ScrollableNative` doesn't exist.